component/install.php

```diff
@@ -522,19 +522,15 @@ private function updateTables() {
 
 		if (!array_key_exists("evaccess", $icols))
 		{
-                    // What is curtent value of sql_mode
-                    $db->setQuery("SELECT @@sql_mode");
-                    $sql_mode = @$db->loadResult();
-                    
                     $db->setQuery("SET SESSION sql_mode=(SELECT REPLACE(@@sql_mode,'NO_ZERO_DATE',''))");
                     @$db->execute();
                     $sql = "ALTER TABLE #__jevents_vevent ADD INDEX evaccess (access)";
                     $db->setQuery($sql);
                     @$db->execute();
-                    
-                    // Return to old value
-                    $db->setQuery("SET SESSION sql_mode=(".$db->quote($sql_mode).")");
-                    @$db->execute();                    
+                    $db->setQuery("SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode,',NO_ZERO_DATE'))");
+                    @$db->execute();
+                    $db->setQuery("SET SESSION sql_mode=(SELECT REPLACE(@@sql_mode,'NO_ZERO_DATE',''))");
```

Comment:
Why are you changing this?

Reply:
It was not my intenttion. It slipped into the PR probably from an out of date repo. Not sure why though as I updated my code before starting working on the fix...

Anyway as far as I can remember that code was changed due to some issues in the sql_mode detection and I remember I worked into that fix. Is my code removing the fix or is it that you didn't merged my fix?

Reply:
I have removed the changes on install script. Please review this PR as it contains a couple of fixes that should be in next release.

Best!!